.secrets

```diff
@@ -5,12 +5,10 @@ ops/ansible/playbooks-ccs/files/client_data_server_local_test_env_test_keypair.p
 ops/ansible/playbooks-ccs/files/client_data_server_local_test_env_prod_keypair.pem
 ops/ansible/playbooks-ccs/files/client_data_server_local_test_env_dpr_keypair.pem
 ops/ansible/playbooks-ccs/vars/prod/group_vars/all/vault.yml
-ops/ansible/playbooks-ccs/vars/prod/group_vars/all/ssh.yml
 ops/ansible/playbooks-ccs/vars/prod-sbx/group_vars/all/vault.yml
-ops/ansible/playbooks-ccs/vars/prod-sbx/group_vars/all/ssh.yml
 ops/ansible/playbooks-ccs/vars/test/group_vars/all/vault.yml
```

Comment:
Could you also drop the remaining `*/vault.yml` files here? I guess I neglected to do so in the initial implementation.

Reply:
Yep, just did so.